mp/src/vgui2/vgui_controls/Label.cpp

```diff
@@ -527,6 +527,7 @@ void Label::Paint()
 				case a_northwest:
 				case a_west:
 				case a_southwest:
+                case a_center:
```

Comment:
Centered text didn't have an offset because it's difficult to determine the offset direction for something completely in the middle. You could argue that it could also subtract the offset as if it were right aligned.

I think it shouldn't have an offset, and instead the xpos should be bumped.

Reply:
Mostly was getting this to work in this way in order to keep the user's bounds in case they want a background on the label.

Reply:
Right, but this is affecting every label of every panel now. This may have far reaching consequences.

Reply:
Very true, ill revert.

Reply:
It may be better to make a subclass of Label that handles drawing two parts of text in one. Speedometer label can then be a subclass of it.